index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require_once 'dbConnect.php';
+require_once 'functions.php';
+
+$db = connectToDb($dbName, $userName, $password);
```

Comment:
$dbName, userName and password should all be defined inside the function - they have the appearence here of appearing out of nowhere so make it feel confusing

Reply:
updated